tests/offset.rs

```diff
@@ -35,9 +35,9 @@ fn tx_offset() {
     let mut tested_output_contract_created_id = false;
     let mut tested_output_recipient = false;
 
-    let cases = 3;
+    let cases = 2;
 
-    TransactionFactory::from_seed(1296)
+    TransactionFactory::from_seed(1295)
```

Comment:
Why are these changed?

Reply:
That is a magic touch indeed, but its just that the factory with a seed `1295` will result in only two transactions covering all the specified cases

Reply:
Maybe add a comment explaining that?

Reply:
https://github.com/FuelLabs/fuel-tx/pull/170/commits/d62afa10d3ba2de8f6496eca42ba0674c21ccedc